tools/SourceKit/tools/sourcekitd/include/sourcekitd/sourcekitd.h

```diff
@@ -309,7 +309,10 @@ typedef enum {
   SOURCEKITD_VARIANT_TYPE_INT64 = 3,
   SOURCEKITD_VARIANT_TYPE_STRING = 4,
   SOURCEKITD_VARIANT_TYPE_UID = 5,
-  SOURCEKITD_VARIANT_TYPE_BOOL = 6
+  SOURCEKITD_VARIANT_TYPE_BOOL = 6,
+  // Not implemented in sourcekitd but in SourceKit
```

Comment:
It's not strictly necessary to sync the same integers here, but it's fine to reserve the integer for a future `SOURCEKITD_VARIANT_TYPE_DOUBLE`.
I'd recommend to have in the comment `Reserved for future addition`.